frontend/document/pdfdocument.lua

```diff
@@ -142,6 +142,9 @@ function PdfDocument:getPageLinks(pageno)
 end
 
 function PdfDocument:saveHighlight(pageno, item)
+    local _, suffix = util.splitFileNameSuffix(self.file)
+    if suffix ~= "pdf" then return end
```

Comment:
`if string.lower(suffix) ~= "pdf"`  probably?

Reply:
Indeed. I wonder where it should go though. (I.e., in this and similar locations or in the function.)

Reply:
Looks like everything related to suffix is done in documentregistry.lua, where I guess that lower(suffix) is done correctly, otherwise we would know :) Elsewhere, we just care about the filename without suffix. So I guess here is fine for this quick workaround.
(Unless you want to support real PDF files with a bofus .tif or .zip extension, anyway opened as a PDF by MuPDF - so we should ask MuPDF for the type of file he has opened :)